geoh5py/io/h5_reader.py

```diff
@@ -248,6 +256,27 @@ def fetch_property_groups(
 
         return property_groups
 
+    @classmethod
+    def fetch_value_map(cls, h5file: Optional[str], uid: uuid.UUID) -> Optional[float]:
```

Comment:
**question:** looks like a copy paste of `fetch_values`. Should return ReferenceValueMap instead?

with check on the type of values?

Reply:
Correct, should return Optional[dict]